cleantalk.php

```diff
@@ -141,6 +141,14 @@
     \Cleantalk\Antispam\DisableComments::getInstance();
 }
 
+// Email encoder
+if (
+    $apbct->key_is_ok &&
```

Comment:
what do you think if you replace such multi-storey conditions with a single function like apbct_is_encoder_on()
You can even create a separate file under these conditions, we have a lot of them

Reply:
Maybe but not sure